client/reader/discover/components/add-new/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { User, UserSelect } from '@automattic/data-stores';
+import { useSelect } from '@wordpress/data';
+import { useTranslate } from 'i18n-calypso';
+import { AddSitesForm } from 'calypso/landing/subscriptions/components/add-sites-form';
+import { SiteSubscriptionsList } from 'calypso/landing/subscriptions/components/site-subscriptions-list';
+import {
+	SubscriptionManagerContextProvider,
+	SubscriptionsPortal,
+} from 'calypso/landing/subscriptions/components/subscription-manager-context';
+
+import './style.scss';
+
+const USER_STORE = User.register();
+
+const DiscoverAddNew = () => {
+	const translate = useTranslate();
+	const isLoggedIn = useSelect(
+		( select ) => ( select( USER_STORE ) as UserSelect ).isCurrentUserLoggedIn(),
+		[]
+	);
+
+	return (
+		<div className="discover-add-new">
+			<SubscriptionManagerContextProvider portal={ SubscriptionsPortal.Reader }>
+				<div className="discover-add-new__form">
+					<h2 className="discover-add-new__form-title">
+						{ translate( 'Add new sites, newsletters, and RSS feeds to your reading list.' ) }
+					</h2>
+					<AddSitesForm onAddFinished={ () => {} } />
```

Comment:
onAddFinished makes me wonder what we are missing, but it seems to send a success message already anyways which is all that came to mind for me. Do we just have to pass an anon function to prevent errors? should the AddSitesForm handle this better and default the prop to a noop so we don't have to pass this in?

On a separate note for tracking events, I see adding through here adds `calypso_subscriptions_site_subscribed` with `source: subscriptions-add-sites-modal`. I wonder if we want to add any other params to differentiate it being added here vs. on subscriptions manage.

Reply:
> onAddFinished makes me wonder what we are missing, but it seems to send a success message already anyways which is all that came to mind for me. Do we just have to pass an anon function to prevent errors? should the AddSitesForm handle this better and default the prop to a noop so we don't have to pass this in?

Yeah, good call on making that prop optional. Done here: https://github.com/Automattic/wp-calypso/pull/99745/commits/16228658d9ebd58034208fd67ec128c5a25841cf. It looks like the prop is [used to close a modal elsewhere](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/subscriptions/components/add-sites-modal/add-sites-modal.tsx#L6).

Reply:
> On a separate note for tracking events, I see adding through here adds calypso_subscriptions_site_subscribed with source: subscriptions-add-sites-modal. I wonder if we want to add any other params to differentiate it being added here vs. on subscriptions manage.

This sounds like a good idea. New issue for it here: https://github.com/Automattic/loop/issues/477